railties/lib/rails/commands/console/console_command.rb

```diff
@@ -31,6 +28,10 @@ def name
       def start
         IRB.setup(nil)
 
+        if !Rails.env.local? && ENV["IRB_USE_AUTOCOMPLETE"] != "true"
```

Comment:
```suggestion
        if !Rails.env.local? && !ENV.key?("IRB_USE_AUTOCOMPLETE")
```

So we don't assume what are valid values.

Reply:
Updated and squashed 👍